README.md

```diff
@@ -218,6 +220,58 @@ Note that First-Party Sets also gives browsers the opportunity to group per-site
 those at `chrome://settings/content/all`) by the “first-party” boundary instead of eTLD+1, which is 
 not always the correct site boundary.
 
+# Clearing Site Data on Set Transitions
+Sites can change which First-Party Set they are a member of. We need to pay attention to these transitions so that they don’t link user identities across all the FPSs they’ve historically been in. In particular, we must ensure that a domain cannot transfer a user identifier from one First-Party Set to another when it changes its set membership.
+
+In order to achieve this, site data needs to be cleared on certain transitions. The clearing should behave like [`Clear-Site-Data: "*"`](https://www.w3.org/TR/clear-site-data/#grammardef-), which includes cookies, storage, cache, as well as execution contexts (documents, workers, etc.). We don’t differentiate between different types of site data because:
+
+ * A user identifier could be stored in any of these storage types.
+ * Clearing just a few of the types would break sites that expect different types of data to be consistent with each other.
+
+Sites switching FPSs can be viewed as “leaving one FPS and joining another”, and the member sites can only add/remove themselves to/from FPSs with the consent from the owner. If a FPS’s owner changes, that set is considered dissolved. Thus we associate the need to clear site data with whether that site’s FPS owner has changed, with a few exceptions.
+
+If a site’s owner changed:
+
+1. If this site is joining a FPS, no site data clearing will be enforced.
+    *   Pro: Avoids adoption pain when a site first joins a FPS.
+    *   Con: Unclear how this lines up with user expectations about access to browsing history prior to set formation.
+2. If this site is leaving a FPS, clear site data of this site.
+
+Potential modification, which adds implementation complexity:
+
+3. If this site’s new owner used to be a member site from the same FPS, no site data clearing will be enforced. 
+    *   Pro: Provides graceful transitions for examples (f) and (g).
+    *   Con: Multi-stage transitions, such as (h) to (i) are unaccounted for.
+
+## Examples
+
+![Clear Site Data Img 1  FPS representation drawio](https://user-images.githubusercontent.com/89418275/137032991-519ab978-b2fe-449b-9563-fda993bee119.png)
+
+---
+
+![Clear Site Data Img 2  FPS joining drawio](https://user-images.githubusercontent.com/89418275/137032021-cb83f31c-27e4-4fc5-b754-d1d920aa5951.png)
+
+1. Site A and Site B create a FPS with Site A as the owner and Site B as the member. Site data will not be cleared.
```

Comment:
The images label themselves with "a)", "b)", etc., but this list is labeled with "1.", "2.", "3.", etc. Make it consistent.